blocks/bricks/__init__.py

```diff
@@ -658,10 +658,17 @@ def add_property(f):
         return add_property
 
     def __getattr__(self, attr):
+        # Prevents infinite recursion when no attribute "f" is available
+        # (happens during serialization under unknown circumstances)
+        try:
+            self_f = self.__getattribute__("f")
```

Comment:
Ah, I see why, but that's because `super(Application, self).__getattribute__(attr)` should really just read `raise AttributeError`. It doesn't make sense to call the `__getattribute__` of the parent class (if `__getattr__` was called it's clear that Python doesn't know where to find the attribute, so we should just end the chaing). So just do `if attr == 'f': return {}` and replace the last line with `raise AttributeError`.